src/pages/app/package/[package].vue

```diff
@@ -31,18 +36,59 @@ async function loadAppInfo() {
       .single()
     app.value = dataApp || app.value
 
-    const date_id = new Date().toISOString().slice(0, 7)
-    const { data } = await supabase
-      .from('app_stats')
-      .select()
+    if (cycleStart && cycleEnd) {
+      const { count: statsCount } = await supabase
+        .from('stats')
+        .select('*', { count: 'exact' })
+        .eq('app_id', id.value)
+        .eq('action', 'set')
+        .gte('created_at', getConvertedDate2(cycleStart))
```

Comment:
can you factorise this code, by not await directly.
then you can add the `gte` with and if and then await all request at once, it will make the page load faster